docs/spl-docs/timechart-command.md

```diff
@@ -0,0 +1,105 @@
+# timechart
+
+## Description
+
+Creates a time series chart with corresponding table of statistics.
+
+A timechart is a statistical aggregation applied to a field to produce a chart, with time used as the X-axis. You can specify a split-by field, where each distinct value of the split-by field becomes a series in the chart. With the `limit` you can specify series filtering. If you set limit=0, no series filtering occurs.
+
+## Syntax
+
+The required syntax is in **bold**.
+
+timechart \
+[limit=\<chart-limit-opt>] \
+[\<bin-options>... ] \
+(**\<single-agg>** [BY **\<split-by-clause>**] )
+
+## Required Arguments
+
+When specifying `timechart` command arguments, `<single-agg>`  BY `<split-by-clause>` is required.
+
+### single-agg
+
+**Syntax:** `count or \<stats-func>(\<field>)` \
```

Comment:
Is this how you want it to render (with the `\`)?
<img width="421" alt="image" src="https://github.com/user-attachments/assets/a963b165-d8d0-4874-9385-648690df85de">

Reply:
No I will change it.